sdk/agentserver/Azure.AI.AgentServer.AgentFramework/src/Azure.AI.AgentServer.AgentFramework.csproj

```
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- Approved override to LtsTargetFramework -->
+    <RequiredTargetFrameworks>$(LtsTargetFramework)</RequiredTargetFrameworks>
+    <TargetFrameworks>$(LtsTargetFramework)</TargetFrameworks>
+
+    <PackageTags>Azure.AI.AgentServer.AgentFramework</PackageTags>
+  </PropertyGroup>
```

```suggestion
    <IsAotCompatible Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net7.0'))">true</IsAotCompatible>
  </PropertyGroup>
```

Azure SDK packages should be AOT-compliant.